source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -1104,7 +1104,8 @@ A monitor clears the connection pool when a server check fails with a network
 or command error (`Network or command error during server check`_).
 When the check fails with a network error it is likely that all connections
 to that server are also closed.
-(See `JAVA-1252 <https://jira.mongodb.org/browse/JAVA-1252>`_).
+(See `JAVA-1252 <https://jira.mongodb.org/browse/JAVA-1252>`_). In this case, a monitor
```

Comment:
"In this case," should say "When the check fails with a network timeout error,"

Reply:
done